arangosh/Utils/ManagedDirectory.cpp

```diff
@@ -637,4 +637,20 @@ ssize_t ManagedDirectory::File::offset() const {
   return fileBytesRead;
 }
 
+void ManagedDirectory::File::skip(size_t count) {
+  // TODO is there a better implementation than just read count bytes?
+  // how does this work with gzip?
+  size_t const bufferSize = 4 * 1024;
+  char buffer[bufferSize]; // alloc on the heap?
```

Comment:
stack allocation should be fine here, because we should never have deep stacks in restore